lib/make_voteable/voter.rb

```diff
@@ -29,7 +29,13 @@ def up_vote(voteable)
           self.down_votes -= 1 if has_attribute?(:down_votes)
         end
       else
-        voting = Voting.create(:voteable => voteable, :voter_id => self.id, :voter_type => self.class.to_s, :up_vote => true)
+        voting = Voting.create(
+          :voteable_type => voteable.class.to_s,
+          :voteable_id => voteable.id,
```

Comment:
@rrciisc hey this is a public repository and that is why you are receiving updates. :)